crates/democracy/src/types.rs

```diff
@@ -17,10 +18,20 @@ pub struct Vote<Balance> {
 }
 
 /// The account is voting directly.
-#[derive(Encode, Decode, Clone, Eq, PartialEq, Default, RuntimeDebug, TypeInfo, MaxEncodedLen)]
-pub struct Voting<Votes> {
+#[derive(Clone, Encode, Decode, Eq, MaxEncodedLen, PartialEq, RuntimeDebug, TypeInfo)]
+#[codec(mel_bound(skip_type_params(MaxVotes)))]
+#[scale_info(skip_type_params(MaxVotes))]
```

Comment:
Ah I see why you initially refactored it. Still, I think this is the better solution